src/w3c/templates/sotd.js

```diff
@@ -330,7 +330,7 @@ function linkToWorkingGroup(conf) {
   }
   return html`
     <p>
-      This document was published by ${[conf.wgHTML]} as ${conf.anOrA}
+      This document was published by ${conf.wgHTML} as ${conf.anOrA}
```

Comment:
Could you do the same for `conf.wgPatentHTML`? Thanks!

Reply:
> Could you do the same for `conf.wgPatentHTML`? Thanks!
I think that it has been done already. please correct if you it somewhere

Reply:
Ah, this time it's just `${[wgPatentHTML]}` without conf 😁

Reply:
And also while you are at it could you also do it for `wgPatentURI`?

Reply:
Sure, I did that already. just about to make the PR